tests/Internal/DateTimeStringBugWorkaroundTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+
+namespace ProophTest\EventStore\Internal;
+
+
+use PHPUnit\Framework\TestCase;
+use Prooph\EventStore\Internal\DateTimeStringBugWorkaround;
+
+class DateTimeStringBugWorkaroundTest extends TestCase
```

Comment:
In this case I think you should write just one test with a `@dataProvider`, not separate methods.

Reply:
Ah, I was not aware of this feature. Rewrite the test and addressed your  other comments.